bindings/nodejs/examples/how_tos/alias_wallet/request-funds.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { Utils, Wallet, initLogger } from '@iota/sdk';
+
+// This example uses secrets in environment variables for simplicity which should not be done in production.
+require('dotenv').config({ path: '.env' });
+
+// Run with command:
+// yarn run-example ./how_tos/alias-wallet/request-funds.ts
+
+// In this example we request funds to an alias wallet.
+async function run() {
+    initLogger();
+    if (!process.env.FAUCET_URL) {
+        throw new Error('.env FAUCET_URL is undefined, see .env.example');
+    }
+    try {
+        const faucetUrl = process.env.FAUCET_URL;
+
+        // Create the wallet
+        const wallet = new Wallet({
+            storagePath: process.env.WALLET_DB_PATH,
+        });
+
+        // Get the account we generated with `create_wallet`
+        const account = await wallet.getAccount('Alice');
+
+        const balance = await account.sync();
+
+        const totalBaseTokenBalance = balance.baseCoin.total;
+        console.log(`Balance before requesting funds on alias address: ${totalBaseTokenBalance}`);
+
+        const aliasId = balance.aliases[0];
+        console.log(`Alias Id: ${aliasId}`);
+
+        // Get Alias address
+        const aliasAddress = Utils.aliasIdToBech32(aliasId, await (await wallet.getClient()).getBech32Hrp());
+        const faucetResponse = await (await wallet.getClient()).requestFundsFromFaucet(faucetUrl, aliasAddress);
+        console.log(faucetResponse);
+
+        await new Promise(resolve => setTimeout(resolve, 5000));
```

Comment:
Often takes a bit longer, no?

Reply:
Worked all the time for me 😆 .
But yeah guess I can make it 10s to be sure. I think I just copied it from somewhere (or maybe Copilot suggested it? :trollface: )